packages/extension-browser/tests/background-script.ts

```diff
@@ -50,7 +49,7 @@ const prepareContentScriptInjection = (sandbox: SinonSandbox, browser: typeof ch
     };
 };
 
-test('It registers for messages.', async (t) => {
+test.serial('It registers for messages.', async (t) => {
```

Comment:
😢

Reply:
This shouldn't be necessary. Something must be missing we need to reset when stubbing globals...

Reply:
Sorry, this wasn't necessary, it was something that I tested and forgot to restore.